provd/systemd/provd.socket

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Socket for Ubuntu Desktop Provisioning Daemon
+
+[Socket]
+ListenStream=/run/provd/provd.sock
+SocketMode=0660
```

Comment:
Did you already figure out what's the preferred way to make the socket rw accessible to the gnome-initial-setup user?